openmetadata-ui/src/main/resources/ui/src/components/dropdown/CheckBoxDropDownList.tsx

```diff
@@ -37,18 +38,21 @@ const CheckBoxDropDownList = ({
               tw-right-0 tw-w-full tw-mt-1 tw-shadow-lg tw-border tw-border-main
               tw-bg-white tw-rounded focus:tw-outline-none"
         role="menu">
-        <div className="py-1" role="none">
+        <div className="tw-py-1" role="none">
           {dropDownList.map((item: DropDownListItem, index: number) =>
             !item.isAdminOnly || isAuthDisabled || isAdminUser ? (
               <div
-                className="tw-cursor-pointer"
+                className="tw-cursor-pointer tw-py-1"
                 key={index}
                 onClick={(e) => onSelect && onSelect(e, item.value as string)}>
                 <input
                   checked={Boolean(
                     selectedItems?.includes(item.value as string)
                   )}
                   className="tw-ml-3 tw-mr-2 tw-align-middle custom-checkbox"
+                  disabled={Boolean(
```

Comment:
```suggestion
                  disabled={disabledItems?.includes(item.value as string)}
```

Reply:
`disabledItems` can be `undefined`, that will set disabled to `undefined`